tests/system_tests_http1_adaptor.py

```diff
@@ -23,67 +23,63 @@
 
 import errno
 import json
+import os
+import re
 import select
 import socket
+from http.client import HTTPSConnection
+from ssl import SSLContext, PROTOCOL_TLS_CLIENT, PROTOCOL_TLS_SERVER
+from ssl import CERT_REQUIRED, SSLSocket
 from time import sleep, time
-from typing import Optional
 from email.parser import BytesParser
-from http.client import HTTPConnection
 
 from proton import Message
 from system_test import TestCase, unittest, main_module, Qdrouterd, QdManager
 from system_test import TIMEOUT, AsyncTestSender, AsyncTestReceiver
-from system_test import retry_exception, curl_available, run_curl
+from system_test import retry_exception, curl_available, run_curl, retry
+from system_test import nginx_available, get_digest, NginxServer
 from http1_tests import http1_ping, TestServer, RequestHandler10
-from http1_tests import RequestMsg, ResponseMsg
+from http1_tests import RequestMsg, ResponseMsg, ResponseValidator
 from http1_tests import ThreadedTestClient, Http1OneRouterTestBase
 from http1_tests import CommonHttp1OneRouterTest
 from http1_tests import CommonHttp1Edge2EdgeTest
 from http1_tests import Http1Edge2EdgeTestBase
 from http1_tests import Http1ClientCloseTestsMixIn
 from http1_tests import Http1CurlTestsMixIn
-from http1_tests import wait_http_listeners_up
+from http1_tests import wait_http_listeners_up, wait_http_listeners_down
 from http1_tests import HttpAdaptorListenerConnectTestBase
 from http1_tests import HttpTlsBadConfigTestsBase
+from http1_tests import http1_simple_request
+from system_tests_ssl import RouterTestSslBase
 
+CA_CERT = RouterTestSslBase.ssl_file('ca-certificate.pem')
```

Comment:
Can we move these constants to system_test.py ? system_tests_tcp_adaptor.py also defines them (https://github.com/skupperproject/skupper-router/blob/main/tests/system_tests_tcp_adaptor.py#L71), so they are now defined in two places.

Once these constants are moved to system_test.py, other tests can be modified to use these constants.

Reply:
I just tried to do that cleanup, and it's way bigger than it looks. First of all, you cannot put them into system_test.py because that causes python to barf on a cyclic include. We'd have to export them from system_tests_ssl.py instead, but getting that file to consistently use the new path variable is a huge change, nevermind the changes to the other TLS tests.

Sounds like a good idea, but this patch is already really big and I'd rather avoid even more changes if it can be helped. This should be done in a follow up change.

Reply:
ok, let's do that in a subsequent commit and keep this commit only to the relevant files.

Reply:
https://github.com/skupperproject/skupper-router/issues/958